pkgs/applications/networking/instant-messengers/arrpc/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, runtimeShell
+, nodejs
+}:
+
+buildNpmPackage {
+  pname = "arrpc";
+  version = "3.2.0";
+
+  src = fetchFromGitHub {
+    owner = "OpenAsar";
+    repo = "arrpc";
+    # Release commits are not tagged
```

Comment:
Can you open an upstream issue asking them to do so?

Reply:
Opened an issue for this: https://github.com/OpenAsar/arrpc/issues/48